ansible_collections/arista/avd/tests/unit/plugins/plugin_utils/eos_validate_state_utils/test_catalog.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+# import pytest
+
+
+class TestCatalog:
+    def test(self):
+        pass
```

Comment:
Add a TODO here, so we can search for it.

Reply:
Fixed in [5eebda8](https://github.com/aristanetworks/ansible-avd/pull/2907/commits/5eebda8f72510128bffa100a3018a8c3b446eb51)